roles/marathon_chronos/templates/chronos-consul.j2

```diff
@@ -0,0 +1,12 @@
+{
+ "service": {
+   "name": "chronos",
+   "tags": [ "chronos" ],
+   "port": 14400,
+   "check": {
+     "script": "curl --silent --show-error --fail --dump-header /dev/stderr --retry 2 --connect-timeout 5 --max-time 5 http://chronos.service.consul:14400/ping",
```

Comment:
Using the chronos.zookeeper.consul dns name in the consul health check might be a problem.

Also, with the packaged version of chronos, creating a consul health check is a bigger challenge. We are looking at just updating the UI to take advantage of the marathon health check instead (CiscoCloud/mantl-ui-frontend#4)